src/main/java/seedu/address/logic/commands/FindPropertyCommand.java

```diff
@@ -16,14 +16,13 @@
     public static final String COMMAND_WORD = "find property";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all properties containing any of "
-            + "the specified keywords (case-insensitive) and/or by giving a price and search for properties "
-            + "with asking prices above or below that amount, then displaying them as a list with index "
-            + "numbers.\n"
-            + "Parameters: [KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT] [t/TYPE] \n"
+            + "the specified keywords (case-insensitive) and/or with the given parameters. \n"
+            + "Parameters: [KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT] [t/TYPE] "
+            + "[a/ADDRESS] [r/REMARKS] [tags/TAGS_SEPARATED_BY_COMMA] [p/POSTAL_CODE] \n"
```

Comment:
just a small nitpick, but personally would prefer the options to be grouped in this way:
`[KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT] [t/TYPE] [p/POSTAL_CODE] [a/ADDRESS] [r/REMARKS] [tags/TAGS_SEPARATED_BY_COMMA]` to make it consistent with the rest of the commands

Reply:
Alright sure.

Reply:
Sorry, this way instead because address should be before postalcode:
`[KEYWORD]... [pl/UPPER_PRICE_LIMIT] [pm/LOWER_PRICE_LIMIT] [t/PROPERTY_TYPE] [a/ADDRESS] [p/POSTAL_CODE] [r/REMARKS] [tags/TAGS_SEPARATED_BY_COMMA]` to make it consistent with the rest of the commands